flycheck.el

```diff
@@ -6182,6 +6183,19 @@ Requires DMD 2.066 or newer.  See URL `http://dlang.org/'."
             (or "Warning" "Deprecation") ": " (message) line-end))
   :modes d-mode)
 
+(flycheck-define-checker elixir-dogma
+  "An Elixir syntax checker using the Dogma analysis tool.
+
+See URL `https://github.com/lpil/dogma/'."
+  :command ("mix" "dogma" "--format=flycheck" source-inplace)
```

Comment:
And last but not least, do we need `source-inplace`, or can we feed the buffer contents to `dogma` via standard input?